### PR TITLE
Add logout test

### PR DESCRIPTION
**Pull request by PhysicsJon**

### CODE REVIEW

File: src/NHSDPublicBrowseAcceptanceTests.Actions/Pages/Homepage.cs [hunk 1]
```diff
@@ -50,5 +50,17 @@ public string LoginLogoutLinkText()
             wait.Until(s => s.FindElement(pages.Homepage.LoginLogoutLink).Displayed);
             return driver.FindElement(pages.Homepage.LoginLogoutLink).Text;
         }
+
+        public void LogOut()
+        {
+            if (LoginLogoutLinkText().Equals("Log out"))
```
Comment (talawson05) [inline src/NHSDPublicBrowseAcceptanceTests.Actions/Pages/Homepage.cs]: Should this be case insensitive?

File: src/NHSDPublicBrowseAcceptanceTests.Tests/Steps/Authorization/Authorization.cs [hunk 2]
```diff
@@ -78,5 +78,25 @@ public void ThenTheUserWillBeInformedTheLoginAttemptWasUnsuccessful(bool email,
...
+        [When(@"the User logs out")]
+        public void WhenTheUserLogsOut()
+        {
+            _test.Pages.Homepage.LogOut();
+        }
+
+        [Then(@"the User is logged out")]
+        public void ThenTheUserIsLoggedOut()
+        {
+            _test.Pages.Homepage.LoginLogoutLinkText().Should().Be("Log in");
```
Comment (talawson05) [inline src/NHSDPublicBrowseAcceptanceTests.Tests/Steps/Authorization/Authorization.cs]: & here, does case matter?

File: src/NHSDPublicBrowseAcceptanceTests.Actions/Pages/Homepage.cs (hunk 1 above)
Comment (PhysicsJon) [reply src/NHSDPublicBrowseAcceptanceTests.Actions/Pages/Homepage.cs]: You are quite correct, fixed in next commit

File: src/NHSDPublicBrowseAcceptanceTests.Tests/Steps/Authorization/Authorization.cs (hunk 2 above)
Comment (PhysicsJon) [reply src/NHSDPublicBrowseAcceptanceTests.Tests/Steps/Authorization/Authorization.cs]: You are quite correct, fixed in next commit

**Review by talawson05 — APPROVED** [review]

**Pull request merged**